app/src/main/java/org/fossasia/susi/ai/skills/skilllisting/SkillListingPresenter.kt

```diff
@@ -79,6 +86,53 @@ class SkillListingPresenter : ISkillListingPresenter,
         skillListingView?.displayError()
     }
 
+    override fun onSkillMetricsFetchSuccess(response: Response<ListSkillMetricsResponse>) {
+        skillListingView?.visibilityProgressBar(false)
+        if (response.isSuccessful && response.body() != null) {
+            Timber.d("METRICS FETCHED")
+            metrics = response.body().metrics
+            if (metrics != null) {
+                metricsList.clear()
+                if (metrics?.rating != null) {
+                    if (metrics?.rating?.size as Int > 0) {
+                        metricsList.add(Pair("SUSI, What are your highest rated skills?", metrics?.rating))
```

Comment:
@iamareebjamal I need to use String resource here, but I am unable to use getString() here nor am I able to get the context here. I searched for it, but I found the use of getString() as we usually do. It would be helpful if you could please help me with this.

Reply:
Usually, I wrap context in an interface and inject it in presenter but dependency injection is not available here

I'd say let it be, or if you do need to, create a `getString` method in view interface

`view.getString(R.string.whatever)`

Reply:
UtilModel too requires context. But the problem is I that I am not able to get the current context in the presenter file. I need to use string resource because this String would need to be translated later on when when I will add the option to change the language of the app locally after changing the UI flow.

Reply:
did u make a constructor ?

Reply:
I simply passed the an object of the SkillsListingFragment which implements the View interface and then I got the context from ```skillsListingFragment.requireContext()``` and then everything is working fine.

Reply:
Then use previously described method